.github/actions/create-android-export-template/action.yaml

```diff
@@ -0,0 +1,28 @@
+name: Create android export template
+description: Creates an android godot export template
+inputs:
+  target:
+    description: Export template's target
+runs:
+  using: composite
+  steps:
+    - name: Setup Godot dependencies
+      uses: ./.github/actions/godot-deps
+    - name: Set up JDK 11
+      uses: actions/setup-java@v1
+      with:
+        java-version: 11
+    - name: Setup Android SDK
+      uses: android-actions/setup-android@v2
+    - name: Download android ${{ inputs.target }} export armv7 binary
+      uses: actions/download-artifact@v3
+      with:
+        name: android-export-${{ inputs.target }}-binary-armv7
+    - name: Download android ${{ inputs.target }} export arm64v8 binary
+      uses: actions/download-artifact@v3
+      with:
+        name: android-export-${{ inputs.target }}-binary-arm64v8
+    - name: Build android ${{ inputs.target }} export template
+      shell: bash
+      run: |
+        cd platform/android/java/ && ./gradlew generateGodotTemplates
```

Comment:
Could we space the steps with a new line for readability (for all the actions) or is this considered best practice?

Reply:
We can, I'll do this in next PR for editor app if it's ok for you ?

Reply:
Ok for me